src/test/java/com/mongodb/kafka/connect/source/topic/mapping/DefaultTopicMapperTest.java

```diff
@@ -42,106 +41,110 @@
 import org.junit.jupiter.params.provider.ValueSource;
 
 import org.bson.BsonDocument;
+import org.bson.BsonString;
 
 import com.mongodb.kafka.connect.source.MongoSourceConfig;
 
 public class DefaultTopicMapperTest {
-
   private static final String PREFIX = "prefix";
   private static final String SUFFIX = "suffix";
-  private static final BsonDocument DB_ONLY_NAMESPACE_DOCUMENT =
-      BsonDocument.parse("{ns: {db: 'db1'}}");
-  private static final BsonDocument NAMESPACE_DOCUMENT =
-      BsonDocument.parse("{ns: {db: 'db1', coll: 'coll1'}}");
-
-  private static final BsonDocument NAMESPACE_ALT_DATABASE_DOCUMENT =
-      BsonDocument.parse("{ns: {db: 'db2', coll: 'coll2'}}");
-
-  private static final BsonDocument NAMESPACE_ALT_COLLECTION_DOCUMENT =
-      BsonDocument.parse("{ns: {db: 'db1', coll: 'coll2'}}");
-
-  private static final String TOPIC_NAMESPACE_MAP =
-      "{\"db1\": \"mappedDBTopic\", \"db1.coll1\": \"mappedDBAndCollTopic\"}";
-
-  private static final String TOPIC_NAMESPACE_ALL_MAP =
-      "{\"*\": \"allTopic\", \"db2.coll2\": \"allExceptionTopic\"}";
+  private static final String TOPIC_NAMESPACE_MAP_EXAMPLE1 =
+      "{'myDb': 'topicTwo', 'myDb.myColl': 'topicOne'}";
+  private static final String TOPIC_NAMESPACE_MAP_EXAMPLE2 =
```

Comment:
Lets test all the template types to ensure they work as expected and help prevent any future regressions.

Reply:
Added test cases for `EXTENSIVE_TOPIC_NAMESPACE_MAP`.

Reply:
Suggest adding cases for: `{sep_coll_sep}` and `{coll_sep}`

Reply:
Added for `sep_coll`, `coll_sep`, `sep_coll_sep`.